### ISSUE TITLE
Error: You are subbmitting a directory, which is not currently supported

### ISSUE THREAD

**Issue by KyleGas** [exercism/exercism#4309]
Hello there.

I am gettin the error that i wrote in the title. I just did my first exercise in javascript. Anybody know what i am doing wrong?

**Comment by rpottsoh** [comment]
indicate the file name of your solution, not just the path.

**Comment by sfairchild** [comment]
Maybe that error message should be updated to help new students who might not have much experience programming yet. 

Maybe something like 
Error: You are subbmitting a directory, which is not currently supported. Please provide the path to the file you wish to submit, example : `exercism submit ./hello_world.js`

**Comment by NobbZ** [comment]
I'm not sure how printing a random file name could help those students that do not know what to do after reading the first sentence...

**Comment by voltadzontra** [comment]
I have almost the same problem with submitting the file. When I write exercism submit FILE_PATH FILE_NAME, it says that  "The file you are trying to submit cannot be found'' athough my file is in the folder. What is the problem?

**Comment by rpottsoh** [comment]
Is the folder the file is in within "exercism workspace"?  The cli only accepts files from within the "workspace" folder.

Sent from my android device.

-----Original Message-----

**Comment by voltadzontra** [comment]
I've found the problem! It was out the workspace! But thanks for the answer!

**Comment by voltadzontra** [comment]
Of course, kytrinyx! The thing is that I made a mistake about my path. So when you copy path, and when you paste it after _exercism submit,_ you must delete "first part of path".

What that means I'll explain on my example: I've installed exercism on C:\Users\[another_folder]\[exercism folder]. When you copy path, you paste whole path, but before hitting an enter, you must delete **C:\Users\[another_folder]\** and than **add in the end your file**.

Final path should look something like this:
exercism submit C:\Users\[another_folder]\Exercism\javascript\hello-world\**hello-world.js**

**Important note!** Make sure that your hello-world.js file is in the folder which you downloaded first! For every exercise you need to download folder first, finish your task, and save your .js file in the folder which you downloaded. That would be your path for submitting.

Feel free to ask if anything you don't understand!

**Comment by kytrinyx** [comment]
This issue was addressed with an update to the error message in https://github.com/exercism/cli/pull/724

**Comment by hmccutchen** [comment]
TLDR- most likely your issue is you only need provise the FILE NAME, NOT the folder and NOT the file path, THE ACTUAL NAME OF THE FILE.

**Comment by elvykiung** [comment]
I did this and it works. You need to be in the hello-world directory and run excercism submit with just the file name like below:

PATH in your terminal --> 'cd  ~/Exercism/hello-world' -> ~/Exercism/hello-world (master)
run 
'$ exercism submit hello-world.js'
